libs/perun/components/src/lib/perun-components.module.ts

```diff
@@ -97,7 +98,11 @@ import { RemoveUserExtSourceDialogComponent } from './remove-user-ext-source-dia
     BugReportDialogComponent,
     AutoFocusDirective,
     UserExtSourcesListComponent,
-    RemoveUserExtSourceDialogComponent
+    RemoveUserExtSourceDialogComponent,
+    AutoFocusDirective,
+    ServicesStatusListComponent,
+    AutoFocusDirective,
+    ServicesStatusListComponent
```

Comment:
you are both importing and exporting twice
also for what is AutoFocusDirective?

Reply:
The double imports and declarations are me messing up rebase.
AutoFocus is Vojtas job, and I guess it should put focus on top element